packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -93,14 +115,38 @@ export default {
 
             this.updateMessage();
 
-            if (modalContext) modalContext.close();
+            if (modalContext) {
+                modalContext.close();
+            }
+        },
+
+        showOrderHistory () {
+            window.location.assign('order-history');
+
+            this.updateMessage();
+        },
+
+        isDuplicateOrderError () {
+            return this.errorCode === 'DuplicateOrder';
+        },
+
+        dataLayerPushDupOrderWarnTrackingEvent () {
```

Comment:
I think this might need to be in the analytics module of the store https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-checkout/src/store/checkoutAnalytics.module.js